packages/components/announcer/src/announcer.ts

```diff
@@ -0,0 +1,79 @@
+import { localized } from '@lit/localize';
+import { ScopedElementsMap } from '@open-wc/scoped-elements/html-element.js';
+import { EventEmitter, EventsController } from '@sl-design-system/shared';
+import { type CSSResultGroup, LitElement, type TemplateResult, html } from 'lit';
+// import { state } from 'lit/decorators.js';
+import { Announcement } from './announcement.js';
+import styles from './announcer.scss.js';
+
+declare global {
+  interface GlobalEventHandlersEventMap {
+    'sl-announce': SlAnnounceEvent;
+    'sl-announce-polite': SlAnnounceEvent;
+    'sl-announce-assertive': SlAnnounceEvent;
+  }
+
+  interface HTMLElementTagNameMap {
+    'sl-announcer': Announcer;
+  }
+}
+
+export type SlAnnounceEvent = CustomEvent<{ message: string; urgency?: 'polite' | 'assertive' }>;
+
+/** Sends a notification to the live aria.
+ * Please be aware that sending messages too soon after each other can cause the screenreader
+ * to stop reading the earlier messages, or even skip them, when a new message is sent.
+ * If you want to send multiple messages in a row, consider using a single message with all the information.
+ *
+ * @param message - The message to send to the live aria.
+ * @param urgency - The urgency of the message. Default is 'polite'.
+ */
+export function sendToAnnouncer(message: string, urgency?: 'polite' | 'assertive'): void {
+  const liveEvent = new EventEmitter<SlAnnounceEvent>(document.body, 'sl-announce');
```

Comment:
No need to do it this way; that is meant more for use in web components. 

`document.body.dispatchEvent(new CustomEvent('sl-announce', { detail: { message, urgency }}));`